src/local.settings.json

```diff
@@ -4,13 +4,15 @@
   "Values": {
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
     "FUNCTIONS_WORKER_RUNTIME": "dotnet",
-    "SourceCosmosDB": "<Your connection string to your Monitored Container database account here>",
-    "SourceDatabase": "<your Monitored Container's database name here>",
-    "SourceCollection": "<your Monitored Container's name here>",
-    "TargetCosmosDB": "<Your connection string to your Target Container databse account here>",
-    "TargetDatabase": "<your Target Container's database name here>",
-    "TargetCollection": "<your Target Container's name here>",
-    "PoisonQueueName": "<name of the Queue to store poison documents>",
-    "PoisonQueueConnectionString": "<Azure Storage Connection String to hold the poison documents Queue>"
+    "SourceCosmosDB": "",
```

Comment:
We might want to leave the previous value here or a comment, not an empty value?

Reply:
yes, I will add.